src/lsp-error-reporter.cpp

```diff
@@ -102,7 +110,9 @@ void lsp_error_formatter::write_before_message(severity sev,
   this->output_.append_decimal_integer(r.end.line);
   this->output_.append_copy(u8",\"character\":");
   this->output_.append_decimal_integer(r.end.character);
-  this->output_.append_copy(u8"}},\"severity\":1,\"message\":\"");
+  this->output_.append_copy(u8"}},\"severity\":");
+  this->output_.append_decimal_integer(severity_value);
```

Comment:
Nit: Writing a character is [probably] cheaper than writing a decimal integer. I'd make severity_value a char8 so we can just append_copy here.